simpleLooping/isPrime.py

```diff
@@ -3,3 +3,12 @@
 # Ex:
 #   isPrime(937453153313) # return True
 #   isPrime(965246846246) # return False
+def isPrime(number: int) -> bool:
+    if number < 2:
+        return False
+    
+    for digit in range(2, int((number ** 0.5) + 1)):
+        if number % digit == 0:
+            return False
+    else:
```

Comment:
What the else doing here? what is it? what the meaning that it adding? can we remove it?